pychunkedgraph/app/segmentation/common.py

```diff
@@ -208,9 +210,14 @@ def handle_root(table_id, atomic_id):
             )
         )
 
+    stop_layer = request.args.get("stop_layer", None)
+    if stop_layer is not None:
+        stop_layer = int(stop_layer)
```

Comment:
Should catch `ValueError` here and return a `BadRequest` exception instead. Uncaught exceptions are [automatically converted to server errors (500)](https://github.com/seung-lab/PyChunkedGraph/blob/a3cece788c3e9fe0fc320d85f6fa975633b7f319/pychunkedgraph/app/segmentation/common.py#L105). But here it's the user input that is incorrect.